bowling/swift/Bowling/README.md

```diff
@@ -14,19 +16,24 @@ In the tenth frame a player who rolls a spare or strike is allowed to roll the e
 balls to complete the frame.  However no more than three balls can be rolled in
 tenth frame.
 
-Original Source: http://butunclebob.com/files/downloads/Bowling%20Game%20Kata.ppt
-
-____
+Sources:
+- [kata logs](http://kata-log.rocks/bowling-game-kata)
+- [PowerPoint](http://butunclebob.com/files/downloads/Bowling%20Game%20Kata.ppt)
 
-1. Write the first failing test. Then write a factory that returns an object, or an aggregate of objects, that make the test pass.
+## How to start:
 
-The factory should be limited to creating objects and linking them together. No conditionals allowed.
+1. Write the first failing test. Then write code to make the first test pass. The code written should only be intended to make the first test pass.
 
 2. Write the next failing test.
 
-3. Can you make it pass by changing the factory and/or creating a new class and nothing else?
+3. Can you make it pass by building over what you already written?
 If yes, great! Go back to 2.
 If not, refactor until you can.
 
-The refactoring should bring the system to a state where it’s possible to implement the next test just by changing the aggregate of objects that is returned by the factory. Be careful not to implement new functionality; the current test should still fail.
+The refactoring should bring the system to a state where it’s possible to implement the next test just by adding the necessary features. Be careful not to implement new functionality; the current test should still fail.
 
+## How to evaluate the dojo:
+- What was the first test? Was it useful? Did they write a good test?
+- What decision (architecture-wise) did they make at the beginning? Did they stick to it? Was it useful (solves the problem) and how simple was it?
+- Method extraction: did they extract any functionalities to outside classes or protocols? If no, why not? Is the code still testable? 
```

Comment:
functionality should be singular here (just to be idiomatic - what you've written makes perfect sense)